recipes/serverfiles/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "serverfiles" %}
+{% set version = "0.2.1" %}
+{% set sha256 = "7dfe1c28c7792231758929e3cbc24bff71bdc0120594d01e6f9b94847addf871" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```

Comment:
Please add `noarch: python`

Reply:
Added.

It might be a good idea to also include the (commented out) option in the `example` recipe